core/src/main/java/com/novoda/merlin/MerlinBuilder.java

```diff
@@ -22,13 +19,13 @@
 
 public class MerlinBuilder {
 
-    private BindListener merlinOnBinder;
-    private ConnectListener merlinConnector;
-    private DisconnectListener merlinDisconnector;
+    private BindCallbackManager merlinOnBinder;
+    private ConnectCallbackManager merlinConnector;
+    private DisconnectCallbackManager merlinDisconnector;
 
-    private MerlinRegisterer<Connectable> connectableRegisterer;
-    private MerlinRegisterer<Disconnectable> disconnectableRegisterer;
-    private MerlinRegisterer<Bindable> bindableRegisterer;
+    private Register<Connectable> connectables;
+    private Register<Disconnectable> disconnectables;
+    private Register<Bindable> bindables;
```

Comment:
Rename to remove verbose naming, we know this is `Merlin` and a `Register` is an official list of something, in this case `Registerable`s.